apps/music/js/metadata/ogg.js

```diff
@@ -97,7 +99,13 @@ var OggMetadata = (function() {
         }
 
         readAllComments(fullpage, metadata);
-        resolve(metadata);
+
+        LazyLoader.load('js/metadata/vorbis_picture.js', function() {
+          VorbisPictureComment.parsePictureComment(metadata).
+            then(function (metadata) {
```

Comment:
Nit: no space between `function` and `(`.

At least, I think that's the standard Gaia style...